test/api/node-after.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
```

Comment:
Note that, as per Contributing.md, you can use --fgrep to run only a subset of WPTs